package.json

```diff
@@ -0,0 +1,218 @@
+{
+  "devDependencies": {
+    "@babel/plugin-transform-modules-commonjs": "^7.23.3",
+    "jest": "^29.7.0",
+    "prettier": "^3.2.2"
+  },
+  "name": "09_javascript_array",
+  "description": "<figure style=\"text-align: center\">    <img src=\"https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/JavaScript-logo.png/640px-JavaScript-logo.png\" alt=\"DOM Tree\" width=\"100\" />    <figcaption>JavaScript</figcaption> </figure>",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "index.js",
+  "dependencies": {
```

Comment:
We only need three devDependencies

"@babel/plugin-transform-modules-commonjs"
"jest"
"prettier"
FIX: remove all dependencies and install only required devDependencies.